resultbrowser/src/css/default.css

```diff
@@ -81,7 +82,7 @@ body {
     background-color: white;
 }
 
-#menu > ul > li {
+#menu>ul>li {
```

Comment:
Please revert this change and all the others that removed the spaces from the child combinator. It's really hard to read selectors formatted like this.

Reply:
Done